app/models/adjustment.rb

```diff
@@ -53,4 +61,20 @@ def storage_locations_belong_to_organization
       errors.add :storage_location, "storage location must belong to organization"
     end
   end
+
+  def replicate_to(itemizable)
```

Comment:
Where are we using this method?

Reply:
:cold_sweat:  um.....

```bash
aaron@toph:~/Documents/Code/rails/diaper$ grep -R replicate_to *
app/models/adjustment.rb:  def replicate_to(itemizable)
aaron@toph:~/Documents/Code/rails/diaper$ 
```

apparently not!

Originally this was used in the magic I was writing at RbtB, but @mdworken and I ended up approaching it from a smarter angle. We can ditch this!